tests/e2e/specs/modules/pagespeed-insights/activation.test.js

```diff
@@ -38,7 +44,7 @@ describe( 'PageSpeed Insights Activation', () => {
 
 		await expect( page ).toMatchElement( 'h2.googlesitekit-setup-module__title', { text: 'PageSpeed Insights' } );
 
-		await page.type( 'input.mdc-text-field__input', 'PSIKEYTOSUBMITANDTEST' );
+		await pasteText( 'input.mdc-text-field__input', 'PSIKEYTOSUBMITANDTEST' );
```

Comment:
This also seems unrelated, unless I'm mistaken.

Reply:
Correct. We have been updating/enhancing older tests with newer utilities as we come back to them.